### DISCUSSION TITLE
24/7 mode command

### DISCUSSION THREAD

**Discussion by Kangurp** [SudhanPlayz/Discord-MusicBot#440] [Ideas]
There's 24/7 mode enabled/disabled forever. Would be cool to modify it by command.

!24/7 - 24/7 mode have been enabled!
!24/7 - 24/7 mode have been disabled!

**Comment by aryan-tah** [comment]
Just make a quick db variable storing true or false and import it from the command file into the botconfig and use it there, such basic commands are not needed as such
